spring-cloud-sleuth-instrumentation/src/main/java/org/springframework/cloud/sleuth/instrument/web/WebFluxSleuthOperators.java

```diff
@@ -94,6 +95,15 @@ private WebFluxSleuthOperators() {
 	 * @param runnable - lambda to execute within the tracing context
 	 */
 	public static void withSpanInScope(Context context, Runnable runnable) {
```

Comment:
@marcingrzejszczak Technically, I could've just changed this parameter to `ContextView` and it wouldn't break applications, but wasn't sure which option you preferred.

Reply:
That's fine